README.md

```diff
@@ -8,38 +8,24 @@ The models can all be used in the same way, using `fit()` and `predict()` functi
 similar to scikit-learn. The library also makes it easy to backtest models,
 and combine the predictions of several models and external regressors.
 
-## install
-
-### using conda
```

Comment:
Should we also keep conda steps somewhere?

Reply:
I see conda setup needs to be checked/rewritten with the addition of pytorch, so maybe a different PR?

issues include:
- it's torch on pip, pytorch on conda -.-
- pytorch actually has some very specific install steps depending on the GPU and on whether using pip vs conda

Reply:
If we have a general way to `conda install u8timeseries` then yes we should document it here. But for a future PR I think.